statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -808,35 +984,44 @@ def __init__(self, coefs, coefs_exog, sigma_u, names=None, _params_info=None):
             self.intercept = np.zeros(self.neqs)
 
     def get_eq_index(self, name):
```

Comment:
This is never actually used.  There's a test in test_var and a commented-out usage in irf.  Remove/privatize?

Reply:
Trying to minimize changes to api now unless I fee strongly about it.